content/docs/getting-started/getting-started-for-admins/index.md

```diff
@@ -71,6 +71,22 @@ By default, GitHub adds your account to the team as a member. You are now an adm
 
 ℹ️  &nbsp;It can take some time for Roadie to refresh the list of teams from GitHub teams. If you do not see admin functions immediately, please wait a few minutes and try again.
 
+### More Information on Permissions Needed
```

Comment:
Can we move this to a dedicated page in the docs? It's only interesting to security minded folk and breaks the flow for others.

I'm imagining a separate page which we link to from here. "Learn more about the permissions required by the GitHub app".